metadata-etl/src/main/java/metadata/etl/ownership/DatasetOwnerEtl.java

```diff
@@ -82,7 +83,9 @@ public void extract() throws Exception {
       channelSftp.put(localFileStream, remoteDir + "/" + JAVA_FILE_NAME + JAVA_EXT, ChannelSftp.OVERWRITE);
       localFileStream.close();
 
-      String hiveQuery = prop.getProperty(Constant.HDFS_OWNER_HIVE_QUERY_KEY);
+      InputStream inputStream = classLoader.getResourceAsStream("hql/fetch_owner.hql");
```

Comment:
since this is a very special use case, please add one line comment to explain that we are going to scan dataset descriptor gathered from HDFS to identify the owner.

fetch_owner_from_dataset_descriptor.sql is a better script name to use. We don't need to use HQL convention anymore.